pkg/controller/testing/table.go

```diff
@@ -290,7 +291,7 @@ func (tc *TestCase) VerifyWantEvent(eventRecorder *MockEventRecorder) error {
 }
 
 func getEventsAsString(events []corev1.Event) []string {
-	eventsAsString := make([]string, 0)
+	eventsAsString := make([]string, len(events))
```

Comment:
`eventsAsString := make([]string, 0, len(events))`

Reply:
Yes, I changed it afterwards. I misread your previous comments. But that should be pushed already

Reply:
Sorry, I must have selected a bad subset of the diffs to view.